gnippy/config.py

```diff
@@ -40,7 +40,7 @@ def get_config(config_file_path=None):
         Returns a dictionary with all the possible configuration options,
         with None for the options that were not provided.
     """
-    if not os.path.isfile(config_file_path):
+    if os.path.isfile(config_file_path):
```

Comment:
This introduces a bug - it'll raise a ConfigFileNotFoundException in the case where a config file is present. Please revert this change.